README.md

```diff
@@ -2,6 +2,5 @@
 * Please **READ CAREFULLY** the instructions
 * Do time yourself (the benchmark is approximately 30 minutes)
 * You may use every resource you would naturally use as a developer, except for AI-based tools such as Co-Pilot, ChatGPT, Bard etc.
-* Please send the completed exercise zipped (.zip, or if the mail blocks just change the extension to something else) to Dror Elovits (d.elovits@tricentis.com) within up to 48h
+* Please send the completed exercise zipped (.zip, or if the mail blocks just change the extension to something else) to Yuval Raz (y.raz@tricentis.com) within up to 48h
```

Comment:
targil bait is fuya

Reply:
Yeah.. not all candidate get there - only the nice ones 😉